app/controllers/concerns/error_handler.rb

```diff
@@ -6,4 +6,7 @@ def render_error(error_code, details = {})
       Error.instance.error_for(error_code, details)
     ), status: Error.instance.http_status_for(error_code)
   end
+
+  class AuthenticationError < StandardError
```

Comment:
Why are you defining this here?

Reply:
This is for cases when the JWT is invalid (this is not new, but was moved to errorHandler module).